### ISSUE TITLE
New InfoBanner component

### ISSUE THREAD

**Issue by martimalek** [freenowtech/wave#192]
### Describe the feature you'd like

Currently **Wave** does not have implemented the `InfoBanner` component. 
The component purpose as described in Figma:

> Vibrant information messages used as page banners or within forms

 This is a component used by at least a couple of teams and it's quite generic so it would be a great addition to the design system.

[Figma designs](https://www.figma.com/file/dTefUItBx5bfbxcCcMzMXm/Wave-DS?node-id=1134%3A24819)

<img width="929" alt="Screenshot 2021-11-11 at 16 44 35" src="https://user-images.githubusercontent.com/46452321/141326720-4848b1c7-8b6d-44ff-8a0f-e6a1c8e8af28.png">

### API
The Props would look something like this.
```typescript
export enum Variants {
    info,
    success,
    warning,
    error,
}
interface InfoBannerProps {
    title: string
    description: string
    variant: Variants
    emphasized: boolean
    linkText?: string
    linkUrl?: string
}
```

**Comment by martimalek** [comment]
I would like opinions on having default props for `variant` and for `emphasized` 🙏🏽

**Comment by rafael-sepeda** [comment]
@martimalek the default variant should be, from my perspective:
`variant: info`
`emphasized: false`
At leas this combination is the most used.

**Comment by nlopin** [comment]
Maybe we can start with one variant and then add on top of that

**Comment by martimalek** [comment]
@rafael-sepeda should we improve the description of the component? At the moment the only text on [the designs](https://www.figma.com/file/dTefUItBx5bfbxcCcMzMXm/Wave-DS?node-id=357%3A28795) is 

> Vibrant information messages used as page banners or within forms

**Comment by rafael-sepeda** [comment]
@martimalek Yes, sure. I actually do not remember where I took this text. If you have any proposals for a better description - let me know.
